en/docs/get-started/overview.md

```diff
@@ -1,95 +1,64 @@
--   Authorization of users
--   Single sign-on
--   Provisioning
--   Access delegation
--   Password reset
--   Self-registration
--   Account locking
-
-### The Identity Server and the solution it provides
-
-WSO2 Identity Server provides secure identity management for enterprise
-web applications, services, and APIs by managing identity and
-entitlements of the users securely and efficiently. The Identity Server
-enables enterprise architects and developers to reduce identity
-provisioning time, guarantee secure online interactions, and deliver a
-reduced single sign-on environment. The WSO2 Identity Server decreases
-the identity management and entitlement management administration burden
-by including the role-based access control (RBAC) convention,
-fine-grained policy-based access control, and Single-Sign-On (SSO)
-bridging.
-
-The Identity Server enables you to create, maintain and terminate user
-accounts along with user identities across multiple systems including
-Cloud applications. When there are multiple applications that
-require authentication, users should be able to log in at one place and
-still have seamless access to all the other applications.
-
-Additionally, the Identity Server brings about a new and improved
-approach to federation. There is a centralized Identity as a Service
-Provider. It is still an overall n to n relationship. There is a 1 to n
-relationship from a federation partner to consumer services (where
-multiple consumer services rely on a single centralized federated
-Identity Provider for security) and a 1 to n relationship from consumer
-service to federation partners (where a single consumer service can rely
-on multiple Identity providers for security). This model ensures greater
-efficiency.
-
-WSO2 Identity Server has the ‘ [Jaggery](http://jaggeryjs.org/) ’ user
-interface for end users. Apart from the [Management
-Console](../../setup/getting-started-with-the-management-console), an [end user
-view](../../learn/user-portal) is available to manage profiles,
-to recover accounts and to manage authorized apps. The log in and
-consent pages in the UI can be completely customized because they run on
-a separate context as a separate web application. These web applications
-can even be dropped into a separate application server if required.
+# Overview 
+
+Are you new to WSO2 Identity Server? Let's get to know about what we are all about! 
+
+WSO2 Identity Server (WSO2 IS) is all about simplifying the Identity and Access Management (IAM) needs of your organization. It is based on open standards and is fully open source. It is released under the [Apache Software License Version 2.0](http://www.apache.org/licenses/LICENSE-2.0).
+
+The product ensures easy integration with a variety of applications to facilitate single sign-on (SSO) using several accepted standards, social login, identity federation, API security, strong authentication, account management, privacy compliance, and more. 
```

Comment:
IMO `several` hints not many but only a few.
Maybe we can have something like (just a suggestion)- `using industry-wide accepted standards`

Reply:
Also can we have this `facilitate single sign-on (SSO) using several accepted standards, social login, identity federation, API security, strong authentication, account management, privacy compliance, and more` in bullet points? WDYT?

Reply:
Yes, bullet points would have been clearer. However, these are the same point we are elaborating in the "why" section. Might look like an unnecessary duplication. WDYT?